config/queue.php

```diff
@@ -39,37 +38,32 @@
             'driver' => 'database',
             'table' => 'jobs',
             'queue' => 'default',
-            'expire' => 60,
```

Comment:
Is this option still looked at in the code?

Reply:
@LukeTowers `expire` was renamed to `retry_after` - `expire` is not used anywhere.